main.cpp

```diff
@@ -1,27 +1,90 @@
 #include "main.h"
 #include "rc.h"
+#include <stdexcept>
+
+int x = 8;
+int y = 8;
+COLORREF color = RGB(33, 33, 33);
+
+COLORREF get_color(HWND parent) {
+	CHOOSECOLOR cc;
+	static COLORREF acrCustClr[16];
+	static DWORD rgbCurrent = RGB(33, 33, 33); 

```

Comment:
Nema potrebe ni za _static_ variablom (to je skrivena globalna), neka funkcija prima trenutni _color_.